File: 1431-25-1749-maximum-absolute-sum-of-any-subarray/1431-25-1749-maximum-absolute-sum-of-any-subarray.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var maxAbsoluteSum = function(nums) {
    let sum = 0,mn = 0,mx = 0
    for(let i = 0 ; i < nums.length;i++){
        sum+=nums[i];
        mx=Math.max(sum,mx);
        mn=Math.min(sum,mn);
    }
    return mx-mn;
};

File: 1431-25-1749-maximum-absolute-sum-of-any-subarray/1431-25-1749-maximum-absolute-sum-of-any-subarray.java
class Solution {
    public int maxAbsoluteSum(int[] nums) {
        int sm = 0,mn =0,mx =0;
        for(int x:nums){
            sm+=x;
            mx=Math.max(mx,sm);
            mn=Math.min(mn,sm);
        }
        return mx - mn;
    }
}